### ISSUE TITLE
Couldn’t save single entry due to validation errors on the slug and/or URI

### ISSUE THREAD

**Issue by p1729** [craftcms/cms#4151]
### Description
uriFormat: __home__ is failing for single entry while project sync 

### Steps to reproduce

1. Create a single section in craft cms 3 and leave the URI blank which creates a single entry in project.yaml with `uriFormat: __home__`
2. Now apply the above changes on your another craft cms instance(staging, dev etc..) via craft 

### Additional info

- Craft version: 3.1.22
- PHP version: 7.3.3
- Database driver & version: MySQL 5.6.0
- Plugins & versions:
   "vlucas/phpdotenv": "^2.4.0"

**Comment by andris-sevcenko** [comment]
Are all the instances created from the same source DB? Project config comparison happens using the UIDs for comparison, so, unless you follow the steps [found here](https://docs.craftcms.com/v3/project-config.html#enabling-the-project-config-file), you'll run into conflicts.

**Comment by p1729** [comment]
all the instances are created from scratch with `craft setup` and all the schema changes are propagated through `project.yaml` from one instance to another. Will this also creates config problem ?

**Comment by andris-sevcenko** [comment]
That should be okay. So there are no sections on the second instance where you're porting changes via  project.yaml in your test case to?

**Comment by p1729** [comment]
yes, I have project.yaml file on git version control along with other folder structure and I tried `craft setup` to create another instance with the project.yaml already having some schema from another instance along with clean DB schema ...but ran into same problem mentioned above...when I tried changing the `uriFormat: __home__` to `uriFormat: '\'` the craft setup got passed for new instances and for old instances project sync also went through and later again I kept the URI blank from the admin login from UI and that got logged in project.yaml as `uriFormat: __home__`

**Comment by proimage** [comment]
I encountered this just now locally; had a Single I couldn't save or delete. Poked around in the DB a bit and found that, unlike most Singles that have just one row in the `craft_entries` table with their `id`, the problematic section had _two_ entries associated with it.

I checked the backend and the ID of one of the entries matched that Single's Entry ID, so I deleted the other one, which fixed the problem.

**Comment by roelvanhintum** [comment]
I had the same issue. It looks like the `titleFormat` was killing for setting up a new environment:
```
name: Home
handle: home
hasTitleField: false
titleLabel: Titel
titleFormat: '{header.headerTitle|striptags}'
```
